code/datums/abilities/flock/flockmind.dm

```diff
@@ -279,6 +302,7 @@
 	for(var/obj/machinery/door/airlock/A in range(10, get_turf(holder.owner)))
 		if(A.canAIControl())
 			targets += A
+	src.tutorial_check(FLOCK_ACTION_GATECRASH, targets, TRUE)
 	if(length(targets))
 		playsound(holder.get_controlling_mob(), 'sound/misc/flockmind/flockmind_cast.ogg', 80, 1)
```

Comment:
```suggestion
	if(length(targets))
		src.tutorial_check(FLOCK_ACTION_GATECRASH, targets, TRUE)
		playsound(holder.get_controlling_mob(), 'sound/misc/flockmind/flockmind_cast.ogg', 80, 1)
```
In case you do it at a distance away?